docs/topics/api/abuse.rst

```diff
@@ -0,0 +1,40 @@
+=============
+Abuse Reports
+=============
+
+The following API endpoint covers abuse reporting
+
+-------------------
+Submitting a report
+-------------------
+
+.. _`abusereport-create`:
+
+The following API endpoint allows an abuse report to be submitted for an Add-on
+or user on addons.mozilla.org.  Authentication is not required, but is recommended
```

Comment:
While reducing the noise in abuse reports would be good, I don't think requiring authentication is the way to go there. We get useful info from anonymous reports, and encouraging account creation is not a goal for AMO.

We do have some ideas to improve on this in the near future, in particular direct support requests, ratings, and abuse reports to where they belong, and have some predetermined abuse report options instead of just an open text field.